atomicapp/nulecule/main.py

```diff
@@ -120,6 +123,56 @@ def __init__(self, app_spec, destination=None,
         self.answers_file = answers_file
         self._process_answers()
 
+    @staticmethod
+    def init(app_name, destination=None, app_version='1.0',
+             app_desc='App description'):
+        """Initialize a new Nulecule app"""
+        context = dict(
+            app_name=app_name,
+            app_version=app_version,
+            app_desc=app_desc,
+            atomicapp_version=__ATOMICAPPVERSION__,
+            nulecule_spec_version=__NULECULESPECVERSION__
+        )
+        tmpdir = tempfile.mkdtemp(prefix='nulecule-new-app-')
+        template_dir = os.path.join(os.path.dirname(__file__),
+                                    '../external/templates/nulecule')
+        if destination is None:
+            destination = os.path.join('.', app_name)
+
+        if os.path.exists(destination) and os.path.isdir(destination) and os.listdir(destination):
+            value = raw_input('Destination directory is not empty! Do you still want to proceed? [Y]/n: ')
+            value = value or 'y'
+            if value.lower() != 'y':
+                print(
+                    '\nAtomic app: %s was not initialized because destination '
+                    'directory: %s is not empty.' % (app_name, destination))
+                return
+
+        distutils.dir_util.copy_tree(template_dir, tmpdir)
+        for item in os.walk(tmpdir):
+            parent_dir, dirs, files = item
+            for filename in files:
+                if not filename.endswith('.tpl'):
+                    continue
+                templ_path = os.path.join(parent_dir, filename)
+                if parent_dir.endswith('artifacts/docker') or parent_dir.endswith('artifacts/kubernetes'):
+                    file_path = os.path.join(
+                        parent_dir,
+                        '{}_{}'.format(app_name, filename[:-4]))
+                else:
+                    file_path = os.path.join(parent_dir, filename[:-4])
+                with open(templ_path) as f:
+                    s = f.read()
+                t = Template(s)
+                with open(file_path, 'w') as f:
+                    f.write(t.safe_substitute(**context))
+                os.remove(templ_path)
+
+        distutils.dir_util.copy_tree(tmpdir, destination, True)
+        distutils.dir_util.remove_tree(tmpdir)
+        print('\nInitialized atomic app: %s at %s' % (app_name, destination))
```

Comment:
Might be better to do this in `cli/main.py` since that is where run/stop/fetch print out information to the user about what happened during execution.